src/main/java/com/google/opensesame/servlets/PersonBuilder.java

```diff
@@ -43,4 +43,55 @@ public PersonBuilder projectIDs(ArrayList<String> projectIDs) {
     this.projectIDs = projectIDs;
     return this;
   }
+
+  /**
+   * Create a PersonObject instance with information from the current builder.
+   *
+   * @return PersonObject that corresponds with feilkds from this PersonBuilder instance
+   * @throws IOException
+   */
+  public PersonObject buildPersonObject() throws IOException {
```

Comment:
Now that you have simplified the constructors for mentors and people using the GitHub API, I wonder if a builder class is still necessary, or if this and the following functions can just be added to the Mentor and People classes. I initally created the builder so that we could have any combination of fields to construct mentors/people, but if we are only ever going to create these objects with a GitHub ID and list of projects/interests, I dont think it is still necessary.

Reply:
Ditto on this. Builder objects add extra complexity, but they give you the flexibility to only set any subset of the possible fields in an object that you could set. If you'll always set the same fields and won't often need to leave any of those fields undefined in your code, it is simpler to just have one constructor.

Reply:
I can see your concerns. The Builder is redundant at the moment but by the next iteration of this PR, I plan to have optional fields in the Person Constructor. I will add a TODO to pivot away from a Builder in the event that It stays pointless by the next iteration of this code.